mlflow_elasticsearchstore/elasticsearch_store.py

```diff
@@ -35,21 +47,34 @@ def _hit_to_mlflow_experiment(self, hit: Any) -> Experiment:
                           artifact_location=hit.artifact_location,
                           lifecycle_stage=hit.lifecycle_stage)
 
-    def _hit_to_mlflow_run(self, hit: Any) -> Run:
+    def _hit_to_mlflow_run(self, hit: Any, use_latest_metrics: bool = False) -> Run:
```

Comment:
You didn't answer the question in my previous comment, when do you need to create a mlflow run with metrics instead of latest metrics ?

Reply:
Maybe never, I will try to remove this